cloud-deploy/README.md

```diff
@@ -147,7 +169,8 @@ environments:
 ```
 
 ```
-service-hours: ## all times in UTC ( scale up at 07:00, scale down to 0 at 22:00)
+service-hours: ## all times in UTC ( scale up at 07:00, scale down to 0 at 22:00)*
```

Comment:
service-hours here but scaling-hours in the example above? Which one is it?

Reply:
scale-hours, updated